site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -1,81 +1,92 @@
 <div id="peer-control">
-    <h1> Options for Peer Matrix </h1>
-        <div class="row">
-            <div class="card col-md-6" style="margin:0px">
-                <div class="content" style="margin:0px">Upload a csv file dictating grader-student pairs for peer grading. The csv file must contain a labelled 'student' column and a 'grader' column: <BR><hr>
-                    <input class="custom-file-input" type="file" name="peer_graders_list" id="peer_graders_list">
-                </div>
+    <h2> Options for Peer Matrix </h2>
```

Comment:
Having the additional spaces surrounding the text in the header strikes me as...weird.
```suggestion
    <h2>Options for Peer Matrix</h2>
```